streams/src/test/java/org/apache/kafka/streams/processor/internals/TasksTest.java

```diff
@@ -52,6 +55,20 @@ public class TasksTest {
 
     private final Tasks tasks = new Tasks(new LogContext());
 
+    @Test
+    public void shouldCheckStateWhenClosedTasks() {
```

Comment:
nit: the name of this unit test is a bit misleading. I would have expected something like `shouldCheckStateWhenRemoveTask()`.

Reply:
Done